### PR TITLE
Feature/better rtfd

### PR DESCRIPTION
**Pull request by myhndl**
Preview: https://polyswarm-client.readthedocs.io/en/feature-better-rtfd

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,37 +1,22 @@
-.. polyswarm-client documentation master file, created by
+.. based on polyswarm-client documentation master file, created by
    sphinx-quickstart on Thu Sep 27 10:11:31 2018.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to the PolySwarm Client Documentation!
-==============================================
+PolySwarm API Source-Based Documentation
```
Comment (mrtizmoatwork) [inline docs/index.rst]: this wording is going to create confusion between polyswarm-client and polyswarm-api. At least say something like "PolySwarm's polyswarm-client API Documentation"

File: docs/conf.py [hunk 2]
```diff
@@ -104,7 +107,7 @@
 # -- Options for HTMLHelp output ---------------------------------------------
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'polyswarm-clientdoc'
+htmlhelp_basename = 'polyswarm-doc'
```
Comment (mrtizmoatwork) [inline docs/conf.py]: how is this used? I could see it being unclear

File: docs/index.rst [hunk 3]
```diff
@@ -1,37 +1,22 @@
...
-==================
+``polyswarm-client`` is a convenient library on which to build PolySwarm market participants.
 
-* :ref:`modindex`
+Here you'll find auto-generated documentation based on the ``polyswarm-client`` source code.
+This may be useful for developers interested in the low-level details of ``polyswarm-client`` implementation.
+Such developers should navigate to "API Reference" over to the left.
 
+Consulting these low level details is unnecessary for developing a successful PolySwarm participant.
+Most users will instead want to consult the `PolySwarm Documentation <https://docs.polyswarm.io>`_.
```
Comment (mrtizmoatwork) [inline docs/index.rst]: This last sentence should be the first sentences. We want to say something to the effect of " Hey if you are just building a particiapnt, go read the docs.. but if you want to know the details of how polyswarm-client works, then keep reading"

File: docs/conf.py (hunk 2 above)
Comment (myhndl) [reply docs/conf.py]: Ah definitely a mistake. I copied work from PolySwarm-api and missed this one

File: docs/index.rst (hunk 1 above)
Comment (myhndl) [reply docs/index.rst]: Oops this was the mistake.

File: docs/conf.py (hunk 2 above)
Comment (myhndl) [reply docs/conf.py]: This one I believe is inconsequential, but I'll revert to be safe.

File: docs/index.rst (hunk 3 above)
Comment (myhndl) [reply docs/index.rst]: Fixed

**Pull request merged**